modules/sandboxed-containers-installing-operator-web-console.adoc

```diff
@@ -3,5 +3,37 @@
 // * sandboxed_containers/deploying_sandboxed_containers.adoc
 
 [id="sandboxed-containers-installing-operator-web-console_{context}"]
+= Installing the {sandboxed-containers-operator} using the web console
 
-= Installing the Sandboxed Containers Operator using the web console
+You can install the {sandboxed-containers-operator} from the {product-title} web console.
+
+.Prerequisites
+
+* You have {product-title} {product-version} installed.
+* You have access to the cluster as a user with the `cluster-admin` role.
+
+.Procedure
+
+. Open a browser window and log in to the {product-title} web console.
+
+. From the *Administrator* perspective, navigate to *Operators* → *OperatorHub*.
+
+. In the *Filter by keyword* field, type `OpenShift sandboxed containers`.
+
+. Select the *{sandboxed-containers-first}* tile.
+
+. Read the information about the Operator and click *Install*.
+
+. On the *Install Operator* page:
+.. Select *preview-1.0* from the list of available *Update Channel* options. This ensures that you install the version of {sandboxed-containers-first} that is compatible with your {product-title} version.
+.. For *Installed Namespace*, ensure that the Operator recommended namespace option is selected. This installs the Operator in the mandatory `openshift-sandboxed-containers-operator` namespace, which is automatically created if it does not exist.
++
+[NOTE]
+====
+Attempting to install the {sandboxed-containers-operator} in a namespace other than `openshift-sandboxed-containers-operator` causes the installation to fail.
+====
+.. For *Approval Strategy*, ensure that *Automatic*, which is the default value, is selected. {sandboxed-containers-first} automatically updates when a new z-stream release is available.
+
+. Click *Install* to make the Operator available to the {sandboxed-containers-first} namespace.
+
+The {sandboxed-containers-operator} is now installed on your cluster. You can trigger the Operator by enabling the runtime on your cluster. You can do this by creating the `KataConfig` custom resource.
```

Comment:
Can we at least add how the KataConfig resource should look like here? 

cc @jensfr @cpmeadors

Reply:
I have been copying the yaml into the console for the console install scenario.  I think it is fair to provide a valid kataconfig yaml for comparison and let the user either use the console and adding the yaml manually or just use the cli.

Reply:
So an simple example, would be:

```
apiVersion: kataconfiguration.openshift.io/v1
kind: KataConfig
metadata:
  name: example-kataconfig
  ```